Makefile

```diff
@@ -18,7 +18,7 @@ TESTS += $(wildcard tests/tangleid/*.sh)
 check: server.py $(DCURL_LIB)
 	@ TMP_PID=`mktemp /tmp/server_pid.XXXXXX`; \
 	echo "Running test suite..." ; \
-	( python $^ & echo $$! > $${TMP_PID} ); \
+	( python3 $^ & echo $$! > $${TMP_PID} ); \
```

Comment:
Follow the sanity check from dcurl to validate the availability of Python3, otherwise it is hard to ensure the consistency.
Check file [mk/python.mk](https://github.com/DLTcollab/dcurl/blob/dev/mk/python.mk) and top-level `Makefile` for details.

Reply:
Don't invoke `python3` directly since package maintainer might not provide the exactly matched executable. Instead, use the detected name from `mk/python.mk`.